[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/528745606",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1767#issuecomment-528745606",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1767",
        "id": 528745606,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyODc0NTYwNg==",
        "user": {
            "login": "lucasjinreal",
            "id": 21303438,
            "node_id": "MDQ6VXNlcjIxMzAzNDM4",
            "avatar_url": "https://avatars.githubusercontent.com/u/21303438?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/lucasjinreal",
            "html_url": "https://github.com/lucasjinreal",
            "followers_url": "https://api.github.com/users/lucasjinreal/followers",
            "following_url": "https://api.github.com/users/lucasjinreal/following{/other_user}",
            "gists_url": "https://api.github.com/users/lucasjinreal/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/lucasjinreal/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/lucasjinreal/subscriptions",
            "organizations_url": "https://api.github.com/users/lucasjinreal/orgs",
            "repos_url": "https://api.github.com/users/lucasjinreal/repos",
            "events_url": "https://api.github.com/users/lucasjinreal/events{/privacy}",
            "received_events_url": "https://api.github.com/users/lucasjinreal/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-09-06T07:30:22Z",
        "updated_at": "2019-09-06T07:30:22Z",
        "author_association": "NONE",
        "body": "```\r\nINFO 09-06 15:25:46 demo_onnxrt_maskrcnn.py:46 - image\r\n[1]    26685 floating point exception  python3 demo_onnxrt_maskrcnn.py\r\n\r\n```",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/528745606/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/529391885",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1767#issuecomment-529391885",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1767",
        "id": 529391885,
        "node_id": "MDEyOklzc3VlQ29tbWVudDUyOTM5MTg4NQ==",
        "user": {
            "login": "skottmckay",
            "id": 979079,
            "node_id": "MDQ6VXNlcjk3OTA3OQ==",
            "avatar_url": "https://avatars.githubusercontent.com/u/979079?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/skottmckay",
            "html_url": "https://github.com/skottmckay",
            "followers_url": "https://api.github.com/users/skottmckay/followers",
            "following_url": "https://api.github.com/users/skottmckay/following{/other_user}",
            "gists_url": "https://api.github.com/users/skottmckay/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/skottmckay/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/skottmckay/subscriptions",
            "organizations_url": "https://api.github.com/users/skottmckay/orgs",
            "repos_url": "https://api.github.com/users/skottmckay/repos",
            "events_url": "https://api.github.com/users/skottmckay/events{/privacy}",
            "received_events_url": "https://api.github.com/users/skottmckay/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-09-09T09:52:25Z",
        "updated_at": "2019-09-09T09:52:25Z",
        "author_association": "MEMBER",
        "body": "I tried using this python code with an arbitrary image from the coco1200 set and the model from the model zoo and it ran successfully. Can you provide details on how ORT was built and maybe the image you're attempting to use?",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/529391885/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/540954126",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1767#issuecomment-540954126",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1767",
        "id": 540954126,
        "node_id": "MDEyOklzc3VlQ29tbWVudDU0MDk1NDEyNg==",
        "user": {
            "login": "nieyan",
            "id": 11882730,
            "node_id": "MDQ6VXNlcjExODgyNzMw",
            "avatar_url": "https://avatars.githubusercontent.com/u/11882730?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/nieyan",
            "html_url": "https://github.com/nieyan",
            "followers_url": "https://api.github.com/users/nieyan/followers",
            "following_url": "https://api.github.com/users/nieyan/following{/other_user}",
            "gists_url": "https://api.github.com/users/nieyan/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/nieyan/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/nieyan/subscriptions",
            "organizations_url": "https://api.github.com/users/nieyan/orgs",
            "repos_url": "https://api.github.com/users/nieyan/repos",
            "events_url": "https://api.github.com/users/nieyan/events{/privacy}",
            "received_events_url": "https://api.github.com/users/nieyan/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-10-11T07:44:14Z",
        "updated_at": "2019-10-11T07:47:09Z",
        "author_association": "NONE",
        "body": "I got the same error using onnxruntime, model was mobilenetv2 which came from onnx model zoo. It seems because of my enviroment, but I can't find out.\r\n\r\n### System information\r\n\r\nOS Platform and Distribution :Linux Ubuntu 18.04(virtual machine)\r\nONNX Runtime installed from :binary(pip install onnxruntime)\r\nONNX Runtime version: 0.5.0\r\nonnx version:  1.6.0                \r\nPython version:3.7.3\r\nVisual Studio version (if applicable):no\r\nGCC/Compiler version (if compiling from source):7.4.0\r\nCUDA/cuDNN version: just CPU\r\nGPU model and memory:just CPU\r\n\r\n### Code for test:\r\n\r\nimport onnxruntime\r\nimport numpy as np\r\nort_session = onnxruntime.InferenceSession('mobilenetv2-1.0.onnx')\r\nX = np.random.random((1, 3, 224, 224)).astype(np.float32)\r\nort_inputs = {ort_session.get_inputs()[0].name: X}\r\nort_outs = ort_session.run(None, ort_inputs)\r\n\r\n### Error message:\r\n[1]    5400 floating point exception (core dumped) ",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/540954126/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/540960472",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/1767#issuecomment-540960472",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/1767",
        "id": 540960472,
        "node_id": "MDEyOklzc3VlQ29tbWVudDU0MDk2MDQ3Mg==",
        "user": {
            "login": "nieyan",
            "id": 11882730,
            "node_id": "MDQ6VXNlcjExODgyNzMw",
            "avatar_url": "https://avatars.githubusercontent.com/u/11882730?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/nieyan",
            "html_url": "https://github.com/nieyan",
            "followers_url": "https://api.github.com/users/nieyan/followers",
            "following_url": "https://api.github.com/users/nieyan/following{/other_user}",
            "gists_url": "https://api.github.com/users/nieyan/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/nieyan/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/nieyan/subscriptions",
            "organizations_url": "https://api.github.com/users/nieyan/orgs",
            "repos_url": "https://api.github.com/users/nieyan/repos",
            "events_url": "https://api.github.com/users/nieyan/events{/privacy}",
            "received_events_url": "https://api.github.com/users/nieyan/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2019-10-11T08:04:10Z",
        "updated_at": "2019-10-11T08:04:10Z",
        "author_association": "NONE",
        "body": "> I got the same error using onnxruntime, model was mobilenetv2 which came from onnx model zoo. It seems because of my enviroment, but I can't find out.\r\n> \r\n> ### System information\r\n> OS Platform and Distribution :Linux Ubuntu 18.04(virtual machine)\r\n> ONNX Runtime installed from :binary(pip install onnxruntime)\r\n> ONNX Runtime version: 0.5.0\r\n> onnx version: 1.6.0\r\n> Python version:3.7.3\r\n> Visual Studio version (if applicable):no\r\n> GCC/Compiler version (if compiling from source):7.4.0\r\n> CUDA/cuDNN version: just CPU\r\n> GPU model and memory:just CPU\r\n> \r\n> ### Code for test:\r\n> import onnxruntime\r\n> import numpy as np\r\n> ort_session = onnxruntime.InferenceSession('mobilenetv2-1.0.onnx')\r\n> X = np.random.random((1, 3, 224, 224)).astype(np.float32)\r\n> ort_inputs = {ort_session.get_inputs()[0].name: X}\r\n> ort_outs = ort_session.run(None, ort_inputs)\r\n> \r\n> ### Error message:\r\n> [1] 5400 floating point exception (core dumped)\r\n\r\nFound virtual machine bring that error. ",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/540960472/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]